napari/_qt/layers/qt_labels_layer.py

```diff
@@ -158,7 +161,15 @@ def __init__(self, layer):
         button_row.setSpacing(4)
         button_row.setContentsMargins(0, 0, 0, 5)
 
+        self.customColorCheckBox = QCheckBox()
```

Comment:
I think here we will want a combo box with the different color modes. We don't expose those right now on the points layer, do we @kevinyamauchi - but do i remember that I saw an example of that on some demo of yours once?

Reply:
We don't currently expose color mode on the Points layer GUI. However, we do so in the [Vectors layer controls](https://github.com/napari/napari/blob/e9df14f6c6f09641faa0e2b41f57a0d55632f7f0/napari/_qt/layers/qt_vectors_layer.py#L70-L75). Adding the functionality to the Points layer is on the road map though.